options.go

```diff
@@ -295,3 +295,13 @@ func WithColorProfile(profile colorprofile.Profile) ProgramOption {
 		p.profile = profile
 	}
 }
+
+// WithInitialSize sets the initial size of the terminal window. This is useful
+// when you have a faulty terminal that doesn't report its size correctly, or
+// if you're testing your program in a non-interactive environment.
+func WithInitialSize(width, height int) ProgramOption {
```

Comment:
Perhaps rename this to `WithWindowSize` to match with WindowSizeMsg 🤔

Reply:
I would say so.